docs/deployment/teams.md

```diff
@@ -53,12 +53,25 @@ Members can't view or perform any actions in the following pages:
 - Projects
 - Settings
 
+### Invite a team member through email
+
+Add team members easily to your organization by sending them an email. This email contains instructions for them to join your org through the same auth provider configured for your account.
+
+You must be an `admin` to perform this operation.
+
+1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep Cloud Platform](https://semgrep.dev/login).
+1. Click **<i class="fa-solid fa-gear"></i> Settings > Access**. This brings you to the **Members** tab.
+1. Click **Invite members**.
+1. In the dialog, enter your team members' email addresses. You can invite up to 20 at a time. Separate each email address with a <kbd>Space</kbd> or <kbd>Tab</kbd> key.
+1. Click **Send invites**.
+1. Repeat this procedure until you have invited all of your team members.
```

Comment:
Hmm. While I'm not positive how we typically onboard large organizations, I would advise against recommending to repeat this flow multiple times.

The limit of 20 was chosen as 2x our free tier limit of 10 contributors.

Reply:
@zzeleznick May I ask why we wouldn't recommend it? I tried to repeat the flow and was successful, if we **don't** want people to onboard 40, 60, etc., then we should probably not allow them, because I envision that they WILL use the invite flow for exactly that even if I don't spell it out.

Reply:
Ideally, we want them to speak to sales!

Reply:
Going to link this conversation to slack to help get more ideas